src/lv_hal/lv_hal_indev.c

```diff
@@ -148,6 +148,11 @@ bool _lv_indev_read(lv_indev_t * indev, lv_indev_data_t * data)
     if(indev->driver.read_cb) {
         LV_LOG_TRACE("indev read started");
         cont = indev->driver.read_cb(&indev->driver, data);
+        if(indev->driver.type == LV_INDEV_TYPE_POINTER) {
+            /* Save these points to be used on the next read call */
+            indev->proc.types.pointer.last_raw_point.x = data->point.x;
+            indev->proc.types.pointer.last_raw_point.y = data->point.y;
+        }
```

Comment:
I think it'd be cleaner to place it at the beginning of `indev_pointer_proc`.

Reply:
Okay. I was trying to keep all uses of `last_raw_point` in one function, but I've updated it as you suggested.